errors/error-handling.js

```diff
@@ -0,0 +1,18 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+    if(err.status && err.message) {
+        console.log(err)
+        res.status(err.status).send({message: err.message})
+    } else {
+        next(err)
+    }
+}
+
+exports.handlePSQLErrors = (err, req, res, next) => {
+    if (err.code === "22P02") {
+        res.status(400).send({message: "invalid input"})
```

Comment:
Usually when there is a 400 error - we would term this as a "Bad Request", rather than "Invalid Input". We could send 400 error codes for many reasons(that may not be anything related to input), and 'Bad Request' is a broader encompassing description.